src/app/routes/index.test.jsx

```diff
@@ -362,78 +341,11 @@ describe('Routes', () => {
       ).toBeInTheDocument();
     });
 
-    it('should route to and render a photo gallery page', async () => {
-      const pathname = '/indonesia/indonesia-41635759';
-
-      fetch.mockResponse(
-        JSON.stringify({
-          ...photoGalleryPageJson,
-          secondaryData: null,
-          recommendations: storyPageRecommendationsData,
-        }),
-      );
-
-      const { getInitialData, pageType } = getMatchingRoute(pathname);
-      const { pageData } = await getInitialData({
-        path: pathname,
-        pageType,
-      });
-      await renderRouter({
-        pathname,
-        pageData,
-        pageType,
-        service: 'indonesia',
-      });
-      const EXPECTED_TEXT_RENDERED_IN_DOCUMENT =
-        'Anies Baswedan, dari mantan menteri menjadi gubernur DKI Jakarta';
-
-      expect(
-        await screen.findByText(EXPECTED_TEXT_RENDERED_IN_DOCUMENT),
-      ).toBeInTheDocument();
-    });
-
-    it('should route to and render a story page', async () => {
```

Comment:
**Note for reviewers:** I have deleted this test because story pages are now routed to the article page